.nvmrc

```diff
@@ -0,0 +1 @@
+16.13.2
```

Comment:
Moving `.nvmrc` to root of repository following https://github.com/guardian/.github/pull/12#discussion_r810084076.